packages/ember-simple-auth/ember-cli-build.js

```diff
@@ -9,11 +9,6 @@ var sourceTrees = [];
 module.exports = function(defaults) {
   var app = new EmberAddon(defaults, {
     storeConfigInMeta: true,
-    'ember-bootstrap': {
-      bootstrapVersion: 4,
-      importBootstrapFont: false,
-      importBootstrapCSS: true
-    },
```

Comment:
why was this removed?

Reply:
It is no longer needed in `packages/ember-simple-auth` since the dummy app has been moved to `packages/test-app` and the dummy app in `packages/ember-simple-auth` is now empty.